supply-chain-security/compromises/2021/php.md

```diff
@@ -0,0 +1,23 @@
+# Compromise of PHP's development team self-hosted Git server
+
+PHP is a web development language primarily used for creating web applications. On March 28th, 2021, they reported [1] that the self-hosted Git server at git.php.net was compromised and included two commits with malicious intent to add backdoor capabilities.
+
+## Impact
+
+* Code changes in the form of two commits to the official and self-hosted Git server were made as if they were signed-off by PHP maintainers Rasmus Lerdorf and Nikita Popov.
+* Code changes were introducing a remote code execution backdoor to any PHP server built from the source-code that included these commits.
+
+## Remediation
+
+* Write access to the repository's source migrated out of a self-built karma system and now restricted only to members of the PHP organization.
```

Comment:
Can we expand on organisation? Is that the GitHub Organisation? Or some other mechanism for identifying developers? If it is GitHub, we should probably move the switch to using GitHub to be the first item in the list.

Reply:
Indeed, will update.

Reply:
Resolved.